### PR TITLE
Rover add proximity (3/3)

### PR DESCRIPTION
**Pull request by khancyr**
Need https://github.com/ArduPilot/ardupilot/pull/6659 and https://github.com/ArduPilot/ardupilot/pull/6659 .

Tested on SITL with fence proximity type!

### CODE REVIEW

File: APMrover2/wscript [hunk 1]
```diff
@@ -22,6 +22,10 @@ def build(bld):
             'AP_VisualOdom',
             'AP_AdvancedFailsafe',
             'AP_WheelEncoder',
+            'AC_Fence',
+            'AP_InertialNav',
+            'AC_Fence',
```
Comment (amilcarlucas) [inline APMrover2/wscript]: Why is AC_Fence twice here ?

File: ArduPlane/wscript [hunk 2]
```diff
@@ -32,7 +32,8 @@ def build(bld):
             'AP_Landing',
             'AP_Beacon',
             'PID',
-            'AP_Soaring'
+            'AP_Soaring',
+            'AC_Fence',
```
Comment (amilcarlucas) [inline ArduPlane/wscript]: This is the only change to Plane. Why ?

File: APMrover2/wscript (hunk 1 above)
Comment (khancyr) [inline APMrover2/wscript]: arf, I haven't rebase on the corrected previous PR. It will be clearer when other will be merged. I will correct that tomorrow

**Comment by khancyr** [comment]
miss the arming check for proximity ...

**Comment by khancyr** [comment]
rebased

**Comment by amilcarlucas** [comment]
Needs rebase :(

**Comment by khancyr** [comment]
replace by https://github.com/ArduPilot/ardupilot/pull/6776

**Pull request closed without merging**